Formula/k/kfr.rb

```diff
@@ -0,0 +1,36 @@
+class Kfr < Formula
+  desc "Fast, modern C++ DSP framework"
+  homepage "https://www.kfrlib.com/"
+  url "https://github.com/kfrlib/kfr/archive/refs/tags/6.1.0.tar.gz"
+  sha256 "551205da23b203daac176a2a76c628e3ad188b9b786e651294bf7ad3e72a9aaf"
+  license "GPL-2.0-or-later"
+
+  depends_on "cmake" => :build
+
+  def install
+    cmake_args = std_cmake_args
+    # C API requires some clang extensions.
+    cmake_args << "-DKFR_ENABLE_CAPI_BUILD=ON" if ENV.compiler == :clang
+
+    system "cmake", "-S", ".", "-B", "build", *cmake_args
```

Comment:
I think this is a little easier to spot the usage of the standard args.
```suggestion
    args = []
    # C API requires some clang extensions.
    args << "-DKFR_ENABLE_CAPI_BUILD=ON" if ENV.compiler == :clang

    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
```